lib/widgets/custom_feeds/group_expressions.dart

```diff
@@ -66,13 +66,23 @@ class _GroupExpressionsState extends State<GroupExpressions> {
                     TwoColumnList(
                       data: _results,
                       useSliver: true,
+                      deleteExpression: (expression) {
+                        context.bloc<PackBloc>().add(
```

Comment:
Again, no need to create this here in constructor but just get Bloc where it's needed actually